data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/http/HttpConsumerVerticleFactory.java

```diff
@@ -157,28 +171,8 @@ private static CircuitBreakerOptions createCircuitBreakerOptions(final DataPlane
     return KafkaProducer.create(vertx, producerConfigs);
   }
 
-  protected KafkaConsumer<String, CloudEvent> createConsumer(
+  private ConsumerRecordSender<String, CloudEvent, HttpResponse<Buffer>> createSender(
```

Comment:
Can you rename it to `createConsumerRecordSender`? It's a long name, i know, but it helps understanding the code here